core-ui/src/main/java/com/readme/android/core_ui/base/BindingActivity.kt

```diff
@@ -38,4 +44,27 @@ abstract class BindingActivity<T : ViewDataBinding>(
 
         return super.dispatchTouchEvent(ev)
     }
+
+    private val mainNavigator: MainNavigator by lazy {
+        EntryPointAccessors.fromActivity(
+            this,
+            Injector.MainNavigaterInjector::class.java
+        ).mainNavigator()
+    }
+
+    private val sharedPreferences: SharedPreferences by lazy {
+        EntryPointAccessors.fromActivity(
+            this,
+            Injector.SharedPreferencesInjector::class.java
+        ).sharedPreferences()
+    }
+
+    fun terminationTokenHandling (viewModel: BaseViewModel){
+        viewModel.moveToLogin.observe(this,EventObserver{
+            mainNavigator.openLogin(this)
+            sharedPreferences.edit().remove("READ_ME_ACCESS_TOKEN").apply()
+            sharedPreferences.edit().remove("USER_NICKNAME").apply()
```

Comment:
너무 보일러 플레이트 코드인 것 같아요.. ktx 활용해서 이렇게 표현하는 게 좋을 것 같아요!
```suggestion
        sharedPreferences.edit { remove("READ_ME_ACCESS_TOKEN") }
        sharedPreferences.edit { remove("USER_NICKNAME") }
```

Reply:
글고 clear 활용해도 괜춘할듯??

Reply:
오케오케 인식